api/src/main/java/org/pmiops/workbench/db/dao/BillingProjectBufferEntryDao.java

```diff
@@ -50,7 +50,7 @@
   @Query(
       "SELECT p.fireCloudProjectName "
           + "FROM BillingProjectBufferEntry p "
-          + "JOIN Workspace w "
+          + "JOIN DbWorkspace w "
```

Comment:
This is a manual fix. Apparently this kind of annotation doesn't read the table name of the `@Table` annotation